.changeset/ninety-kangaroos-doubt.md

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
can we add a new test case for mjs loader

Reply:
done~